tests/test_cluster.py

```diff
@@ -1989,6 +2025,29 @@ def test_persistent_security_group(self):
                                                     }
                                                 }})
 
+    def test_bastion_node_public_dns(self):
```

Comment:
@vogxn : I have added the test suggested by you. You can find it [here](https://github.com/sourabh912/qds-sdk-py/commit/3917d830ea6e7468212265a8f40dd1456ccde9fa).